From: Ben Greiner <code@bnavigator.de>
Date: 2020-05-02 16:33:59 +0200
Subject: Disable the network part of test_embed_svg_url
Upstream: no

OBS can't access the wikimedia url. Disabling this part of the test.

---
diff -u -r ipython-7.14.0orig/IPython/core/tests/test_display.py ipython-7.14.0/IPython/core/tests/test_display.py
--- ipython-7.14.0orig/IPython/core/tests/test_display.py	2020-05-02 16:33:59.393228712 +0200
+++ ipython-7.14.0/IPython/core/tests/test_display.py	2020-05-02 16:35:11.825203757 +0200
@@ -106,10 +106,11 @@
         svg = display.SVG(url=url + 'z')
         nt.assert_true(svg._repr_svg_().startswith('<svg'))
 
-    # do it for real: 6.1kB of data
-    url = "https://upload.wikimedia.org/wikipedia/commons/3/30/Vector-based_example.svg"
-    svg = display.SVG(url=url)
-    nt.assert_true(svg._repr_svg_().startswith('<svg'))
+    # no network on OBS
+    # # do it for real: 6.1kB of data
+    # url = "https://upload.wikimedia.org/wikipedia/commons/3/30/Vector-based_example.svg"
+    # svg = display.SVG(url=url)
+    # nt.assert_true(svg._repr_svg_().startswith('<svg'))
     
 def test_retina_jpeg():
     here = os.path.dirname(__file__)
